pubsub/pubsub.go

```diff
@@ -164,8 +164,14 @@ func (m *Message) Ack() {
 }
 
 // Nack tells the server that this Message was not processed and should be
-// redelivered. It returns immediately, but the actual nack is sent in the
-// background, and is not guaranteed to succeed.
+// redelivered (negative acknowledgement). It returns immediately, but the
+// actual nack is sent in the background, and is not guaranteed to succeed.
+//
+// Nack is a performance optimization for retrying transient failures: it
```

Comment:
"transient failures. It"

Reply:
Not digging the multiple colon-delimited phrases? 😆 Done.